bin/test-lxd-images

```diff
@@ -234,6 +234,12 @@ for url in $(lxc query "/1.0/instances" | jq -r .[]); do
 
     # Systemd cleanliness
     if lxc exec "${name}" -- sh -c "type systemctl" >/dev/null 2>&1; then
+        # Ensure systemctl is ready. If not, it returns "Failed to connect to bus".
+        for _ in $(seq 20); do
+            lxc exec "${name}" -- systemctl >/dev/null 2>&1 && break
+            sleep 0.5
+        done
+
```

Comment:
I came across this while doing manual tests (failing Linux Mint images are a false positive). However, it would run `systemctl --failed` right after launching which caused this issue.

If we are waiting anyway, this "fix" shouldn't be needed.

Reply:
Yeah, the test script has a `sleep 1m` after we're done launching all instances so this shouldn't be needed.